code/ModelAdmin.php

```diff
@@ -238,6 +250,77 @@ public function getExportFields()
         return singleton($this->modelClass)->summaryFields();
     }
 
+    /**
+     * @deprecated 5.0
+     * @return \SilverStripe\ORM\Search\SearchContext
+     */
+    public function getSearchContext($context = null)
```

Comment:
I don't think you need `$context` as a param? It wasn't in the API that was removed. If we do need it, we should add a `@param`.

Reply:
I think it was a leftover from a previous solution. Removed.